Cargo.toml

```diff
@@ -13,6 +13,7 @@ halfbrown = "0.1"
 page_size = "0.4"
 itoa = "0.4"
 ryu = "1"
+core_arch = "0.1.5"
```

Comment:
not really necessary, just using it temporarily (also locally patched until it's released again - there is a known issue where mod.rs for core_arch needs to be patched to include feature, I think `stmt_expr_attributes`)